view/frontend/web/js/view/payment/method-renderer/omise-offline-tesco-method.js

```diff
@@ -0,0 +1,88 @@
+define(
+    [
+        'jquery',
+        'ko',
+        'mage/storage',
+        'Magento_Checkout/js/view/payment/default',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/action/redirect-on-success',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/url-builder',
```

Comment:
@jacekstan guess we don't need `Magento_Checkout/js/model/url-builder` here anymore?